dev/Repeater/SelectionModelChildrenRequestedEventArgs.cpp

```diff
@@ -18,6 +19,11 @@ winrt::IInspectable SelectionModelChildrenRequestedEventArgs::Source()
     return m_source.get();
 }
 
+winrt::IndexPath SelectionModelChildrenRequestedEventArgs::SourceIndex()
+{
+    return m_sourceNode->IndexPath();
```

Comment:
>m_sourceNode [](start = 11, length = 12)

Should we check to make sure m_sourceNode isn't null first? There is nothing here which prevents the args from being created with a nullptr sourceNode.  Alternatively we should probably throw with a reasonable error message if we ever create an event args with a null sourceNode.